pkg/cmd/admin/prune/images.go

```diff
@@ -27,9 +28,19 @@ import (
 )
 
 const (
-	imagesLongDesc = `%s %s - prunes images`
 	// PruneImagesRecommendedName is the recommended command name
 	PruneImagesRecommendedName = "images"
+
+	imagesLongDesc = `Prune images no longer needed by the system due to age and status.
+
+Only a user with a cluster role %s or higher who is logged-in will be able to actually delete
+the images.`
```

Comment:
Resulting help message looks good:

    $ oadm prune images --help
    Prune images no longer needed by the system due to age and status.

    Only a logged in user with a cluster role system:image-pruner or higher will be able to
    actually delete the images.

    Usage:
      oadm prune images [options]

And I let myself inspire once again by looking at help message in other admin command.